deploy/charts/rook-ceph/values.yaml

```
@@ -579,6 +579,8 @@ csi:
 
 # -- Enable discovery daemon
 enableDiscoveryDaemon: false
+# -- Set the discovery daemon device discovery interval (default to 60m)
+discoveryDaemonInterval:
```

Let's set the default so it's obvious what the default is
```suggestion
discoveryDaemonInterval: 60m
```